mesop/server/server.py

```diff
@@ -270,3 +277,15 @@ def handle_message(message):
     flask_app.socketio = socketio  # type: ignore
 
   return flask_app
+
+
+def get_static_folder() -> str | None:
+  if not app_config.static_folder:
```

Comment:
if static_folder is ".", then I think a user could end up serving their entire mesop app directory as a static asset. I think we should add some kind of validation (probably both here and in config.py) to make sure that users don't configure bad static folder paths